python_modules/dagster-graphql/dagster_graphql/schema/pipelines/pipeline.py

```diff
@@ -644,12 +646,15 @@ def resolve_modes(self, _graphene_info: ResolveInfo):
             )
         ]
 
-    def resolve_solid_handle(self, _graphene_info: ResolveInfo, handleID):
-        return build_solid_handles(self.get_represented_pipeline()).get(handleID)
+    def resolve_solid_handle(
+        self, _graphene_info: ResolveInfo, handleID: str
+    ) -> GrapheneSolidHandle:
+        return build_solid_handles(self.get_represented_pipeline()).get(handleID)  # type: ignore  # (unclear if this can return None)
```

Comment:
[1] this can return None - pretty sure field is nullable (not explicitly non-nullable)